SPECS/libdb/libdb.spec

```diff
@@ -67,12 +67,13 @@ rm -rf %{buildroot}
 %{_includedir}/*
 
 %changelog
+* Tue Aug 03 2021 Nicolas guibourge <nicolasg@microsoft.com> - 5.3.28-5
+- Nopatch CVE-2017-3604 to CVE-2017-3617 and CVE-2020-2981
```

Comment:
looks like we missed the 
`PatchX: CVE-XXXX.nopatch`

Reply:
having these files fail the build because their size is below 13 bytes and rpmbuild conplains about that